<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1790</width>
    <height>1110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Tango GUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(236, 236, 236)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label_pmac_polling">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>5</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TANGO minimum polling delay:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="io_tango_polling">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>5</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: solid;

border-bottom-color: #777777;
border-bottom-width: 1px;

border-right-color:#777777;
border-right-width: 1px;

border-top-color:#AAAAAA;
border-top-width: 1px;

border-left-color:#AAAAAA;
border-left-width: 1px;

 background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);</string>
    </property>
    <property name="text">
     <string>250</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_TANGO_pollingDelay">
    <property name="geometry">
     <rect>
      <x>858</x>
      <y>3</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: solid;

border-bottom-color: #777777;
border-bottom-width: 2px;

border-right-color:#777777;
border-right-width: 2px;

border-top-color:#AAAAAA;
border-top-width: 1px;

border-left-color:#AAAAAA;
border-left-width: 1px;

border-radius: 3px;
padding: 1px;
 background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);</string>
    </property>
    <property name="text">
     <string>Set new polling delay</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pmac_pollingvalue">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>5</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>250 ms</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="label_timeedit">
    <property name="geometry">
     <rect>
      <x>1256</x>
      <y>4</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss.zzz</string>
    </property>
    <property name="currentSectionIndex">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_pmac_lastpolling">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>5</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TANGO last polling at:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_TANGO_polling">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>3</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: solid;

border-bottom-color: #777777; border-bottom-width: 2px;

border-right-color:#777777;
border-right-width: 2px;

border-top-color:#AAAAAA;
border-top-width: 1px;

border-left-color:#AAAAAA;
border-left-width: 1px;

border-radius: 3px;
padding: 1px;
 background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);</string>
    </property>
    <property name="text">
     <string>Start polling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_polling">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TANGO polling status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TANGO_polling">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>5</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>236</green>
          <blue>236</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>inactive</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
